Isabelle/Chi_Calculus_Examples/Diamond.thy

```diff
@@ -237,7 +224,8 @@ proof -
     using context_localization sorry
   also have "\<currency>\<^sup>*a \<parallel> b \<triangleright>\<^sup>\<infinity> x. (\<currency>\<^sup>+a \<parallel> a \<triangleleft> x) \<approx>\<^sub>\<flat> \<currency>\<^sup>*a \<parallel> b \<triangleright>\<^sup>\<infinity> x. a \<triangleleft> x"
     using context_localization sorry
-  finally show ?thesis .
+  finally show ?thesis
+    sorry
```

Comment:
Why doesn’t `.` work anymore?

Reply:
Suddenly `.` stopped working, I guess it has to do with the new definitions.

Reply:
This seems to be the same kind of issue I discussed [above](https://github.com/input-output-hk/fm-ouroboros/pull/131#discussion_r310252126).